tests/io_utils.go

```diff
@@ -45,32 +46,84 @@ func NodeNameWithHandler() string {
 	return node.ObjectMeta.Name
 }
 
-func ExecuteCommandInVirtHandlerPod(nodeName string, args []string) error {
+func ExecuteCommandInVirtHandlerPod(nodeName string, args []string) (string, error) {
 	virtClient, err := kubecli.GetKubevirtClient()
 	if err != nil {
-		return err
+		return "", err
 	}
 
 	pod, err := kubecli.NewVirtHandlerClient(virtClient).Namespace(flags.KubeVirtInstallNamespace).ForNode(nodeName).Pod()
 	if err != nil {
-		return err
+		return "", err
 	}
 
 	stdout, stderr, err := ExecuteCommandOnPodV2(virtClient, pod, "virt-handler", args)
 	if err != nil {
-		return fmt.Errorf("Failed excuting command=%v, error=%v, stdout=%s, stderr=%s", args, err, stdout, stderr)
+		return "", fmt.Errorf("Failed excuting command=%v, error=%v, stdout=%s, stderr=%s", args, err, stdout, stderr)
 	}
-	return nil
+	return stdout, nil
+}
+
+func CreateErrorDisk(nodeName string) (string, string) {
+	By("Creating error disk")
+	args := []string{"/usr/bin/virt-chroot", "--mount", "/proc/1/ns/mnt", "exec", "--", "/usr/sbin/modprobe", "scsi_debug", "opts=2", "every_nth=4"}
+	_, err := ExecuteCommandInVirtHandlerPod(nodeName, args)
+	ExpectWithOffset(1, err).NotTo(HaveOccurred(), "Failed to create faulty disk")
+
+	args = []string{"/usr/bin/virt-chroot", "--mount", "/proc/1/ns/mnt", "exec", "--", "/usr/bin/lsscsi"}
+	stdout, err := ExecuteCommandInVirtHandlerPod(nodeName, args)
+	ExpectWithOffset(1, err).NotTo(HaveOccurred(), "Failed to find out address of  faulty disk")
+
+	lines := strings.Split(stdout, "\n")
+	address := ""
+	device := ""
+	for _, line := range lines {
+		if strings.Contains(line, "scsi_debug") {
+			line = strings.TrimSpace(line)
```

Comment:
Maybe a comment with an example on how the output looks like fort this line would help the reader understand what's going on here.